chrome/nav.js

```diff
@@ -221,8 +228,12 @@ function onGameMessage( event ) {
 	if ( config.missionDisplay ) {
 		chrome.storage.local.get( [ ukey + 'mlist' ], showMissions );
 	}
-
-	configured = true;
+    if ( config.displayVisited ) {
+        chrome.storage.local.get( [ ukey + 'visit' ], highlightVisited );
+    } else {
+        chrome.storage.local.remove( [ ukey + 'visit' ] );
```

Comment:
so the tiles never get removed from storage?

at least ... not until the user goes to sweetener (1) -> options (2) -> nav & minimap (3) -> untick highlighting (4) -> retick highlighting (5) -> close tab (6) -> refresh nav (7) -> which is 7 clicks at minimum to "clear" the history